### PR TITLE
First paragraph edit

### PR DESCRIPTION
**Pull request by 50m3rer0**
Updated first paragraph for clarity

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by 50m3rer0** [comment]
Suggested change to 1st paragraph for easier read flow

File: source/_docs/backend/database.markdown [hunk 1]
```diff
@@ -3,9 +3,9 @@ title: "Database"
 description: "Details about the database used by Home Assistant."
 ---
 
-Database is used in by Home Assistant as history and tracker only, to store the events and its parameters. The default database used by Home Assistant is [SQLite](https://www.sqlite.org/), and the database file is stored in your [configuration directory](/getting-started/configuration/) (e.g., `<path to config dir>/home-assistant_v2.db`). If you prefer to run a database server (e.g.,  PostgreSQL), use the [`recorder` component](/integrations/recorder/).
+Home Assistant uses databases to store events and parametersis for history and tracking. The default database used is [SQLite](https://www.sqlite.org/) and the database file is stored in your [configuration directory](/getting-started/configuration/) (e.g., `<path to config dir>/home-assistant_v2.db`); however, other databases can be used. If you prefer to run a database server (e.g.,  PostgreSQL), use the [`recorder` component](/integrations/recorder/).
```
Comment (frenck) [inline source/_docs/backend/database.markdown]: Home Assistant uses 1 database for this purpose, so using plural seems wrong in this case.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/backend/database.markdown (hunk 1 above)
Comment (50m3rer0) [reply source/_docs/backend/database.markdown]: Thanks for the clarification. I'll be happy yo change "uses databases" to "uses a database", but I'm not sure how to do that yet (i'm looking at Hacktoberfest's beginners resources to find out)

Comment (frenck) [reply source/_docs/backend/database.markdown]: image

Go to the "Files changed" tab -> hit the `...` menu on the file you want to edit -> Edit file

**Review by frenck — APPROVED** [review]

**Pull request merged**